lib/modules/hosts/derpibooru.js

```diff
@@ -0,0 +1,38 @@
+addLibrary('mediaHosts', 'derpibooru', {
+	name: 'Derpibooru',
+	domains: ['derpiboo.ru'],
+	matchRe: /^https?:\/\/(?:www\.)?derpiboo.ru\/\d+$/i,
```

Comment:
RES is only for modern browsers: last few versions of Chrome, Firefox, OperaBlink, and last two or three versions of Safari. Safari 6 and Opera 12 get a token nod.